ui/src/views/404/index.jsx

```diff
@@ -1,5 +1,52 @@
 // @flow
+import { colour } from 'pasteup';
+import Logo from 'assets/images/guardian-logo-320.svg';
+import Navigation from './Navigation';
+import Footer from './Footer';
 
-import style from './style.js.scss';
+import {
+    heading,
+    fluidWrap,
+    topbar,
+    subHeadingContainer,
+    subHeading,
+    para,
+} from './style.js.scss';
 
-export default () => <div style={style.message}>404</div>;
+export default () =>
+    <div style={fluidWrap}>
+        <div style={topbar}>
+            <a
+                href="/"
+                style={{
+                    color: '#005689',
+                    textDecoration: 'none',
+                    fontFamily: 'arial',
+                    padding: '8px 10px 7px',
+                    display: 'block',
+                }}>
+                Home
+            </a>
+        </div>
+        <Logo
```

Comment:
> As an example, I think its good to know you are using the logo as an svg vs using the logo as an svg with a fallback

can you explain what it is that you would prefer about that?

i guess this setup argues that abstracting is good, and that constructing the DOM/patching it is only a platform requirement, best passed to the machine to manage as much as we can. in this sense, it prioritises domain/business problems over implementation details.

in the case of a svg, I would argue we already think this way – it very uncommon to see the entire DOM of an svg in HTML source code. we wrap the various elements of an SVG up in a separate file and include that as necessary. 

i think that same paradigm is at play here, just writ larger. at the point we use svg (in the current templates), we don't need to know if the paths are efficient. but we do want to be able to check them at some point, make amendments etc then forget it.

if we are comfortable with that use of an svg – we don't want to hard code all the nodes and don't mind how they're arranged etc, as long as it's done well – are you able to put your finger on what it is about this level of abstraction – we don't know what format the image is, how well the desired format falls back etc, just that it will render a logo – that bothers you?

i feel like its the same principle – do you see a difference in them?